dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3390,9 +3390,6 @@ button.button-active-no-border:active {
   // an active button. (overriding button:active rule above)
   border: none !important;
 }
-button:hover {
```

Comment:
Should we log a ticket to remove all the `box-shadow: none` that won't be needed anymore?

Reply:
mmm that feels like something that would be destined to sit in the backlog forever :). Also crosses teams, might be a little slow/painful to figure out if any are actually needed to override some other styling, and the utility feels relatively low (I think the main improvement here is not needing to add any new override styling). What do you think?

Reply:
Yeah - good points and makes sense. cc: @hannahbergam who mentioned this in the [Slack thread](https://codedotorg.slack.com/archives/C046G4TRLEN/p1707844459615579) you referenced above. Thanks!